### PR TITLE
add qBittorrent icon

### PR DESCRIPTION
**Pull request by jlapitan**
**Issue:**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by jlapitan** [comment]
@birjolaxew please review changes

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3639,6 +3644,6 @@
             "title": "Zulip",
             "hex": "52C2AF",
             "source": "https://github.com/zulip/zulip/"
-        }
+        }
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Just to not litter the commit history, can you remove the whitespace at the end of this line?

```suggestion
        }
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -3639,6 +3644,6 @@
             "title": "Zulip",
             "hex": "52C2AF",
             "source": "https://github.com/zulip/zulip/"
-        }
+        }        
     ]
-}
+}
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Just to not litter the commit history, can you revert the change here (i.e. add a final newline to the file)?

```suggestion
}

```

**Comment by jlapitan** [comment]
> Thanks for the update @jlapitan, that is already looking a lot better! I still have two small requests for changes regarding whitespace in the data file. Also, care to tell us why you picked `#2F67BA` as a colour?

I select that color based on their background color and it is the same value on their main css file on their website

**Comment by ericcornelissen** [comment]
> I select that color based on their background color and it is the same value on their main css file on their website

Okay :+1: If you commit my last suggestion, we can merge this in :slightly_smiling_face:
